modules/objdetect/src/face_detect.cpp

```diff
@@ -200,7 +200,11 @@ class FaceDetectorYNImpl : public FaceDetectorYN
                     float score = std::sqrt(cls_score * obj_score);
                     face.at<float>(0, 14) = score;
 
-                    // Get bounding box
+                    // Checking if the score meets the threshold before adding the face
+                    if (score >= scoreThreshold) 
+                        continue;
+                    
+                    //get bounding box
```

Comment:
Formatting issue:

```
// Get bounding box
```

Reply:
So, Shall i remove that commented line?
Also can you tell me why is there a formatting issue? 

> Formatting issue:
> 
> ```
> // Get bounding box
> ```